File: public/phase-2-20160620022203/C/greedy-man.c
// Name : Greedy Man
#include <stdio.h>
int main() {
	// input:
	// m : 
	// n : 
	// mn : 
	int m, n, mn[1000], ouput=8, index;
	scanf("%d %d", &m, &n);
	for(index = 0; index< m; index++) {
		scanf("%d", &mn[index]);
	}


	// write your code here
	// store your results in `ouput`

	// output
	printf("%d", ouput);
	return 0;
}

File: public/phase-2-20160620022203/JAVA/greedy-man.java
// Name : Greedy Man
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// m : 
		// n : 
		// mn : 
		int index;
		Scanner in = new Scanner(System.in);
		String[] elements = (in.nextLine()).split(" ");
		int m = Integer.parseInt(elements[0]);
		int n = Integer.parseInt(elements[1]);
		int[] mn = new int[m];
		String[] mn_elements = (in.nextLine()).split(" ");
		for(index=0;index<m;index++)
			mn[index] = Integer.parseInt(mn_elements[index]);


		// write your code here
		// store your results in `ouput`

		// output
		// Dummy Data
		int ouput=8;

		System.out.println(ouput);

	}
}